docs/html/_sources/dev_guide/testing.rst.txt

```diff
@@ -0,0 +1,86 @@
+Testing E3SM Diagnostics
+==============================================
+
```

Comment:
I will add an anchor link here to set up the developer environment on #390.

Something like "Make sure to have your developer environment setup locally and on the test server."